client/api_wrapper.go

```diff
@@ -0,0 +1,26 @@
+package client
+
+import (
+	rpcapi "github.com/nspcc-dev/neofs-api-go/v2/rpc"
+	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+	"github.com/nspcc-dev/neofs-api-go/v2/session"
+)
+
+// apiWrapper hides github.com/nspcc-dev/neofs-api-go/v2/rpc] from Client.
+//
+// apiWrapper should be extended on demand.
+type apiWrapper interface {
+	sessionAPI
+}
+
+// sessionAPI should contain only session operations. Dividing interfaces should help to improve testability.
+type sessionAPI interface {
```

Comment:
`neoFSAPIServer` not suitable for this?

Reply:
Yes, looks the same. `neoFSAPIServer` has comment - interface of NeoFS API server. Exists for test purposes only. I think it isn't a true and it using in real client communication. Should I remove the comment about `test purposes only`?

Reply:
I tend to thing, I've understood the comment wrong, for the first time. I will replace my wrapper with this one

Reply:
replaced

Reply:
> interface of NeoFS API server. Exists for test purposes only

just in case, I will say that this phrase means the pointlessness of using the interface without the need for testing (the need to override the network interface)